src/main/java/com/djrapitops/extension/DKBansExtension.java

```diff
@@ -0,0 +1,401 @@
+            iconColor = Color.AMBER,
+            showInPlayerTable = true
+    )
+    public long bans(UUID playerUUID) {
+        return getBanSystem().getPlayerManager().getPlayer(playerUUID).getStats().getBans();
+    }
+
+    @NumberProvider(
+            text = "Unbans",
+            description = "How many unbans the player has on DKBans",
+            priority = 24,
+            iconName = "calendar-check",
+            iconFamily = Family.REGULAR,
+            iconColor = Color.AMBER,
+            showInPlayerTable = true
+    )
+    public long unbans(UUID playerUUID) {
+        return getBanSystem().getPlayerManager().getPlayer(playerUUID).getStats().getUnbans();
+    }
+
+    @NumberProvider(
+            text = "Mutes",
+            description = "How many mutes the player has on DKBans",
+            priority = 23,
+            iconName = "volume-mute",
+            iconFamily = Family.SOLID,
+            iconColor = Color.AMBER,
+            showInPlayerTable = true
+    )
+    public long mutes(UUID playerUUID) {
+        return getBanSystem().getPlayerManager().getPlayer(playerUUID).getStats().getMutes();
+    }
+
+    @NumberProvider(
+            text = "Warnings",
+            description = "How many warnings the player has on DKBans",
+            priority = 22,
+            iconName = "exclamation",
+            iconFamily = Family.SOLID,
+            iconColor = Color.AMBER,
+            showInPlayerTable = true
+    )
+    public long warnings(UUID playerUUID) {
+        return getBanSystem().getPlayerManager().getPlayer(playerUUID).getStats().getWarns();
+    }
+
+    @NumberProvider(
+            text = "Reports",
+            description = "How many reports the player has on DKBans",
+            priority = 21,
+            iconName = "exclamation",
+            iconFamily = Family.SOLID,
+            iconColor = Color.AMBER,
+            showInPlayerTable = true
+    )
+    public long reports(UUID playerUUID) {
+        return getBanSystem().getPlayerManager().getPlayer(playerUUID).getStats().getReports();
+    }
+
+    @PercentageProvider(
+            text = "Report accept ratio",
+            description = "The accepted/denied ratio for reports on DKBans",
+            priority = 20,
+            iconName = "exclamation",
+            iconFamily = Family.SOLID,
+            iconColor = Color.AMBER,
+            showInPlayerTable = true
+    )
+    public double reportPercentage(UUID playerUUID) {
+        PlayerStats stats = getBanSystem().getPlayerManager().getPlayer(playerUUID).getStats();
+        long accepted = stats.getReportsAccepted();
+        return calculatePercentage(stats.getReportsDenied() + accepted, accepted);
+    }
+
+    // Server
+
+    @NumberProvider(
+            text = "Bans",
+            description = "Total amount of bans on DKBans",
+            priority = 25,
+            iconName = "flag",
+            iconColor = Color.AMBER
+    )
+    public long bans() {
```

Comment:
Server methods are not called because matching `CallEvents` or manual calls to `Caller` are not present